src/ToolWindowManager.cpp

```diff
@@ -647,9 +647,14 @@ void ToolWindowManager::restoreState(const QVariantMap &dataMap)
   }
 }
 
-ToolWindowManagerArea *ToolWindowManager::createArea()
+ToolWindowManagerArea *ToolWindowManager::createArea(QWidget *owner)
 {
-  ToolWindowManagerArea *area = new ToolWindowManagerArea(this, 0);
+  if(nullptr == owner)
```

Comment:
```suggestion
  if(owner == NULL)
```

non-yoda comparisons please :).

Reply:
Whoops, sorry about that. personal style slipped in!